Modules/mmapmodule.c

```diff
@@ -1244,8 +1253,8 @@ new_mmap_object(PyTypeObject *type, PyObject *args, PyObject *kwdict)
                                 "tagname",
                                 "access", "offset", NULL };
 
-    if (!PyArg_ParseTupleAndKeywords(args, kwdict, "in|ziL", keywords,
-                                     &fileno, &map_size,
+    if (!PyArg_ParseTupleAndKeywords(args, kwdict, "in|ziL:mmap.__new__",
```

Comment:
why is it related to bpo-28261?

ISTM that currently the error messages are inconsistent:
```
>>> mmap.mmap(-1, -1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OverflowError: memory mapped length must be postiive
>>> mmap.mmap(-1, -1 << 1000)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OverflowError: Python int too large to convert to C ssize_t
```
the patches change it to:
```
>>> mmap.mmap(-1, -1)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OverflowError: memory mapped length must be non-negative
>>> mmap.mmap(-1, -1 << 1000)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
OverflowError: mmap() argument out of range
```